content/en/database_monitoring/setup_oracle/troubleshooting.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Troubleshooting DBM Setup for Oracle
+kind: documentation
+description: Troubleshoot Database Monitoring setup for Oracle
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+This page details common issues with setting up and using Database Monitoring with Oracle, and how to resolve them. Datadog recommends staying on the latest stable Agent version and adhering to the latest [setup documentation][1], as it can change with Agent version releases.
+
+## Common issues
+
+### "Connection refused" error
+Check the connectivity between the Agent and the monitored database by running one of these commands on the machine where the Agent is running, then investigating any errors:
+
+- `nc -v <DB_SERVER> <PORT>`
+- `curl <DB_SERVER>:<PORT>`
+- `telnet <DB_SERVER> <PORT>`
+
+It's important to specify the exact values for `<DB_SERVER>` and `<PORT>` that are configured for that instance in the `oracle-dbm` configuration file.
+
+Using the `telnet` command as an example, the expected output for a correctly configured connection is
+
+{{< code-block lang="text" disable_copy="true" collapsible="true" >}}
+Trying <DB_SERVER_IP_ADDRESS>...
+Connected to <DB_SERVER_NAME>.
+Escape character is '^]'.
+{{< /code-block >}}
```

Comment:
@nenadnoveljic What sorts of errors might the user see here? Will they know which corresponding actions to take?

This portion seems more like guidance for internal troubleshooting practices.

Reply:
> What sorts of errors might the user see here? 

The error messages produced by these utilities are typically self-explanatory. It's reasonable to assume that even less technically proficient support staff should be able to understand them. If it turns out that this isn't the case, we can always improve the clarity.

> Will they know which corresponding actions to take?

These utilities primarily check network connectivity, and a broken connection can result from various reasons, including network or operating system issues, firewall configurations, agent setup errors (e.g., incorrect host or port settings), or issues with the listener, among others. The recommended actions to address these issues can be diverse and specific to the problem at hand.

> This portion seems more like guidance for internal troubleshooting practices.

Why do you believe this guidance is intended solely for internal use? I believe these instructions have universal applicability and can be used in various contexts. If a customer opens a support ticket and reports an error message, our initial response would likely involve instructing them to verify network connectivity using one of the listed commands. In the worst case, this could save a support interaction, and in the best case, the customer may not need to open a ticket at all if they realize that the issue doesn't originate from Datadog.